prototype/src/js/ZoneTour.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+const images = require('../img/*');
+const classNames = require('classnames');
+
+export default class ZoneTour extends React.Component {
+    render() {
```

Comment:
Probably not worth changing but for components without declared lifecycle methods we typically use functional components and destructure their props in the arguments list, like:

```
export default function ZoneTour({
    zone: {
        tour: [
            {
                image,
                caption,
            },
        ],
...etc
```

Reply:
OK cool. Will leave as is, since it's just a prototype. But will remember for the future. Lesson learned!